config.go

```diff
@@ -20,13 +30,62 @@ type RegistryConfig struct {
 	// Map of index host to Basic auth string (base64 encoded
 	// username:password), to make it easy to copypasta from docker
 	// config.
-	Auths map[string]struct {
-		Auth string `json:"auth" yaml:"auth"`
-	} `json:"auths" yaml:"auths"`
+	Auths map[string]Auth `json:"auths" yaml:"auths"`
+}
+
+type Auth struct {
+	Auth string `json:"auth" yaml:"auth"`
 }
 
 type InstanceConfig struct {
 	Git      GitConfig      `json:"git" yaml:"git"`
 	Slack    SlackConfig    `json:"slack" yaml:"slack"`
 	Registry RegistryConfig `json:"registry" yaml:"registry"`
 }
+
+func (c InstanceConfig) HideSecrets() InstanceConfig {
+	c.Git = c.Git.HideKey()
+	for host, auth := range c.Registry.Auths {
+		c.Registry.Auths[host] = auth.HidePassword()
+	}
+	return c
+}
+
+func (a Auth) HidePassword() Auth {
+	if a.Auth == "" {
+		return a
+	}
+	bytes, err := base64.StdEncoding.DecodeString(a.Auth)
```

Comment:
Because that's what the docker config does, for better or worse, and I chose to accept creds in that form to make it easier to supply them via clipboard (or just pointing at the file).